encoding/text/testdata/txt.capnp

```diff
@@ -55,7 +55,7 @@ const mapVal @0xb167974479102805 :Value = (map = [
   (key = "foo", value = (void = void)),
   (key = "bar", value = (void = void)),
 ]);
-const data @0x8e85252144f61858 :Value = (data = 0x"4869 dead beef cafe");
+const data @0x8e85252144f61858 :Value = (data = 0x"4869 dead beef cafe 22275c");
```

Comment:
Same here.  Can you just explain like I'm 5 what the `22275c` corresponds to and why it's needed?

Reply:
This is just the hex encoding of the characters added above.